### ISSUE TITLE
`make install` fails on Arch Linux

### ISSUE THREAD

**Issue by slester** [cryptoparty/handbook#117]
I followed the instructions and installed all of the prerequisites, but `make install` just hung. I removed `>/dev/null` and got the following:

```
This is pdfTeX, Version 3.14159265-2.6-1.40.15 (TeX Live 2014/Arch Linux) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./build/tex/main.tex
LaTeX2e <2014/05/01>
Babel <3.9l> and hyphenation patterns for 79 languages loaded.
(/usr/share/texmf-dist/tex/latex/koma-script/scrbook.cls
Document Class: scrbook 2015/02/07 v3.16 KOMA-Script document class (book)
(/usr/share/texmf-dist/tex/latex/koma-script/scrkbase.sty
(/usr/share/texmf-dist/tex/latex/koma-script/scrbase.sty
(/usr/share/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/share/texmf-dist/tex/latex/koma-script/scrlfile.sty
Package scrlfile, 2015/02/07 v3.16 KOMA-Script package (loading files)
                  Copyright (C) Markus Kohm

))) (/usr/share/texmf-dist/tex/latex/koma-script/tocbasic.sty)
(/usr/share/texmf-dist/tex/latex/koma-script/scrsize11pt.clo)
(/usr/share/texmf-dist/tex/latex/koma-script/typearea.sty
Package typearea, 2015/02/07 v3.16 KOMA-Script package (type area)
                  Copyright (C) Frank Neukam, 1992-1994
                  Copyright (C) Markus Kohm, 1994-

)) (/usr/share/texmf-dist/tex/latex/base/fontenc.sty
(/usr/share/texmf-dist/tex/latex/base/t1enc.def))
(/usr/share/texmf-dist/tex/latex/amsmath/amsmath.sty
For additional information on amsmath, use the `?' option.
(/usr/share/texmf-dist/tex/latex/amsmath/amstext.sty
(/usr/share/texmf-dist/tex/latex/amsmath/amsgen.sty))
(/usr/share/texmf-dist/tex/latex/amsmath/amsbsy.sty)
(/usr/share/texmf-dist/tex/latex/amsmath/amsopn.sty))
(/usr/share/texmf-dist/tex/latex/amsfonts/amssymb.sty
(/usr/share/texmf-dist/tex/latex/amsfonts/amsfonts.sty))
(/usr/share/texmf-dist/tex/latex/base/latexsym.sty)
(/usr/share/texmf-dist/tex/latex/algorithms/algorithm.sty
(/usr/share/texmf-dist/tex/latex/float/float.sty)
(/usr/share/texmf-dist/tex/latex/base/ifthen.sty))
(/usr/share/texmf-dist/tex/latex/algorithms/algorithmic.sty)
(/usr/share/texmf-dist/tex/latex/graphics/graphicx.sty
(/usr/share/texmf-dist/tex/latex/graphics/graphics.sty
(/usr/share/texmf-dist/tex/latex/graphics/trig.sty)
(/usr/share/texmf-dist/tex/latex/latexconfig/graphics.cfg)
(/usr/share/texmf-dist/tex/latex/pdftex-def/pdftex.def
(/usr/share/texmf-dist/tex/generic/oberdiek/infwarerr.sty)
(/usr/share/texmf-dist/tex/generic/oberdiek/ltxcmds.sty))))
(/usr/share/texmf-dist/tex/latex/tools/varioref.sty)
(/usr/share/texmf-dist/tex/latex/hyperref/hyperref.sty
(/usr/share/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty
(/usr/share/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty))
(/usr/share/texmf-dist/tex/generic/ifxetex/ifxetex.sty)
(/usr/share/texmf-dist/tex/latex/oberdiek/auxhook.sty)
(/usr/share/texmf-dist/tex/latex/oberdiek/kvoptions.sty)
(/usr/share/texmf-dist/tex/latex/hyperref/pd1enc.def)
(/usr/share/texmf-dist/tex/latex/latexconfig/hyperref.cfg)
(/usr/share/texmf-dist/tex/latex/url/url.sty))

Package hyperref Message: Driver (autodetected): hpdftex.

(/usr/share/texmf-dist/tex/latex/hyperref/hpdftex.def
(/usr/share/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty))
(/usr/share/texmf-dist/tex/latex/paralist/paralist.sty)
(/usr/share/texmf-dist/tex/latex/eurosym/eurosym.sty)
(/usr/share/texmf-dist/tex/latex/placeins/placeins.sty)
(/usr/share/texmf-dist/tex/latex/pdfpages/pdfpages.sty
(/usr/share/texmf-dist/tex/latex/tools/calc.sty)
(/usr/share/texmf-dist/tex/latex/eso-pic/eso-pic.sty
(/usr/share/texmf-dist/tex/latex/xcolor/xcolor.sty
(/usr/share/texmf-dist/tex/latex/latexconfig/color.cfg)))
(/usr/share/texmf-dist/tex/latex/pdfpages/pppdftex.def))
(/usr/share/texmf-dist/tex/latex/tocloft/tocloft.sty)
(/usr/share/texmf-dist/tex/latex/upquote/upquote.sty
(/usr/share/texmf-dist/tex/latex/base/textcomp.sty
(/usr/share/texmf-dist/tex/latex/base/ts1enc.def)))

LaTeX Warning: Unused global option(s):
    [oribibl].

No file main.aux.
(/usr/share/texmf-dist/tex/latex/base/ts1cmr.fd)
(/usr/share/texmf-dist/tex/context/base/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
) (/usr/share/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
(/usr/share/texmf-dist/tex/latex/oberdiek/grfext.sty)
(/usr/share/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))
(/usr/share/texmf-dist/tex/latex/hyperref/nameref.sty
(/usr/share/texmf-dist/tex/generic/oberdiek/gettitlestring.sty))
<cover.pdf, id=4, 597.23125pt x 845.1575pt> <use cover.pdf> <use cover.pdf>
<cover.pdf, id=7, page=1, 597.23125pt x 845.1575pt> <use cover.pdf, page 1>
<use cover.pdf, page 1> <use cover.pdf, page 1> <use cover.pdf, page 1>
<use cover.pdf, page 1> [1 <./cover.pdf>] [2{/var/lib/texmf/fonts/map/pdftex/up
dmap/pdftex.map}] (/usr/share/texmf-dist/tex/latex/base/t1cmss.fd) [3] [4]
Chapter 1.

Class scrbook Warning: \float@addtolists detected!
(scrbook)              Implementation of \float@addtolist became
(scrbook)              deprecated in KOMA-Script v3.01 2008/11/14 and
(scrbook)              has been replaced by several more flexible
(scrbook)              features of package `tocbasic`.
(scrbook)              Since Version 3.12 support for deprecated
(scrbook)              \float@addtolist interface has been
(scrbook)              restricted to only some of the KOMA-Script
(scrbook)              features and been removed from others.
(scrbook)              Loading of package `scrhack' may help to
(scrbook)              avoid this warning, if you are using a
(scrbook)              a package that still implements the
(scrbook)              deprecated \float@addtolist interface .

(build/tex/chapter_00_introducing_cryptoparty/about_this_book.md.tex
(/usr/share/texmf-dist/tex/latex/base/t1cmtt.fd)
(/usr/share/texmf-dist/tex/latex/amsfonts/umsa.fd)
(/usr/share/texmf-dist/tex/latex/amsfonts/umsb.fd)
(/usr/share/texmf-dist/tex/latex/base/ulasy.fd)
! Undefined control sequence.
<recently read> \tightlist 
                           
l.37 \tightlist
               
? 
```

If I spam hitting Enter, it ends up failing

```
Makefile:22: recipe for target 'latex-pdf' failed
make: *** [latex-pdf] Error 1
```

**Comment by slester** [comment]
pandoc version: 1.15.0.6
pdfTeX 3.14159265-2.6-1.40.15 (TeX Live 2014/Arch Linux)

**Comment by yuvadm** [comment]
Same here. Works on `pandoc-bin` from AUR currently outdated on version 1.13.2. When upgrading to 1.15.0.6 I get the same freeze/error.

**Comment by samthetechie** [comment]
I am not sure how to fix that, maybe someone that knows pandoc well can
replicate the bug and work it out? :)

When upgrades are not upgrades...

**Comment by slester** [comment]
I saw elsewhere that `pandoc` uses `\tightlist` for something now, which `pdflatex` doesn't know what to do with.  Seems `pandoc` assumes inclusion of [its default template](https://github.com/jgm/pandoc-templates/blob/master/default.latex). That may be worth noting in the future. I'll make a PR with the temporary fix.

**Comment by yuvadm** [comment]
Fixed by #118
